aiondemand/AIOD-rest-api
# Docs/schema change

## Change

Adds documentation on how to change the schema when adding an attribute to an existing entity.

## How to Test

Make sure you install the documentation dependencies (`pip install -e ".[docs]"` or refer to `pyproject.toml`).
Read and follow the documentation, see if it works, makes sense, and is intuitive.

## Checklist
  - No unintended changes have been committed.
  - The changes in isolation seem reasonable.
  - Anything that may be odd or unintuitive is provided with a GitHub comment explaining it (but consider if this should not be a code comment or in the documentation instead).

## Related Issues


See comment below.
